### ISSUE TITLE
Implement basic mutations 

### ISSUE THREAD

**Issue by syoon123** [audiopedia-t4sg/api#12]
After #9  is done, implement mutations for
    - [ ] creating/updating a topic (title, audio url)
    - [ ] creating/updating a playlist (title, audio url, topic)
    - [ ] bulk creating tracks (title, audio url, transcript optional, playlist)
    - [ ] updating a track by id
    - [ ] deleting a topic/playlist/track
    - [ ] "removing"/archiving a topic/playlist/track
   
Language not accounted for in these specs; if you are confident that having a Language model is sufficient/good practice you can implement that and the mutations accordingly, otherwise we can add that in a future migration.

Create a new issue for other mutations you think will be necessary.
